updater/lib/dependabot/service.rb

```diff
@@ -100,21 +101,19 @@ def update_dependency_list(dependency_snapshot:)
     end
     def capture_exception(error:, job: nil, dependency: nil, dependency_group: nil, tags: {}, extra: {})
       ::Dependabot::OpenTelemetry.record_exception(error: error, job: job, tags: tags)
-      T.unsafe(Raven).capture_exception(
+      ::Sentry.capture_exception(
         error,
-        {
-          tags: tags.merge({
-            "gh.dependabot_api.update_job.id": job&.id,
-            "gh.dependabot_api.update_config.package_manager": job&.package_manager,
-            "gh.repo.is_private": job&.repo_private?
-          }.compact),
-          extra: extra.merge({
-            dependency_name: dependency&.name,
-            dependency_group: dependency_group&.name
-          }.compact),
-          user: {
-            id: job&.repo_owner
-          }
+        tags: tags.merge({
+          "gh.dependabot_api.update_job.id": job&.id,
+          "gh.dependabot_api.update_config.package_manager": job&.package_manager,
+          "gh.repo.is_private": job&.repo_private?
+        }.compact),
+        extra: extra.merge({
+          dependency_name: dependency&.name,
+          dependency_group: dependency_group&.name
+        }.compact),
+        user: {
+          id: job&.repo_owner
```

Comment:
`capture_exception` now expects `kwargs` instead of a `Hash`. See https://github.com/getsentry/sentry-ruby/blob/7666c94f1f2671e4c42cd8c4a6991e3ce7da56f9/sentry-ruby/lib/sentry-ruby.rb#L394-L401